invisible_cities/reco/paolina_functions_test.py

```diff
@@ -469,4 +475,47 @@ def test_voxel_is_dropped(hits, requested_voxel_dimensions):
     drop_voxel(voxels, voxel_to_drop)
 
     assert len(voxels) == n_starting_voxels - 1
+
+
+@given(bunch_of_hits, box_sizes, min_n_of_voxels)
+def test_total_energy_is_conserved(hits, requested_voxel_dimensions, min_voxels):
+
+    tot_initial_energy = sum(h.E for h in hits)
+    voxels = voxelize_hits(hits, requested_voxel_dimensions, strict_voxel_size=False)
+    energies = [v.E for v in voxels]
+    e_thr = random.uniform(min(energies), max(energies))
```

Comment:
Generating random data inside Hypothesis properties (and keeping Hypothesis out of the loop) makes test failures irreproducible and therefore destroys Hypothesis' ability to shrink counterexamples, and present you with the simplest failing case. You really don't want to do this! Use `hypothesis.strategies.random_module` or `hypothesis.strategies.randoms`.

Reply:
Done.